_includes/nav.html

```diff
@@ -0,0 +1,95 @@
+<ul class="primary">
+    {% for item in site.data.pages %}
+
+        {% if item.categories %}
+
+            <li class="has-children {% if page.url == item.url %}is-active{% endif %}">
+                <a href="{{ item.url | relative_url }}" data-overviewtitle="{{ item.title }} Overview">{{ item.title }}</a>
+                
+                {% for cat in item.categories %}
+                    <div class="category">
+                        <h6>{{ cat.category }}</h6>
+                        {% if cat.pages %}
+                            <ul>
+                                {% for pg in cat.pages %}
+                                    {% if pg.subpageitems %}
+                                        <li class="{% if page.url == pg.url %}is-active{% endif %}">
+                                            <a href="{{ pg.url | relative_url }}"{% if pg.target %} target="{{ pg.target }}"{% endif %}>{{ pg.title }}</a>
```

Comment:
If target could be `_blank` and it's an external URL this must use `rel="noopener referrer"` to avoid leaking the `window.opener` object for security. See https://mathiasbynens.github.io/rel-noopener/ for more information.

This kind of thing could really use a unit test so that we check the output, it would be easy to overlook when content gets updated.

Reply:
I've added the noopener referrer attribute to the template, so it doesn't need to be remembered when updating content.